microsoft-edge/devtools-guide-chromium/rendering-tools/js-runtime.md

```diff
@@ -32,56 +32,119 @@ Record exactly which functions were called and how much memory each requires wit
 Visualize your profiles as a flame chart.
 
 
+<!-- ====================================================================== -->
+## Use the "Allocation sampling" profiling type to view memory allocation by JavaScript function
+
+To use the **Allocation sampling** profiling type to view memory allocation by JavaScript function:
+
+1. Open a webpage; for example, open the [Example 3: Scattered objects](https://microsoftedge.github.io/Demos/devtools-memory-heap-snapshot/example-03.html) demo in a new window or tab.
+
+1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS).  DevTools opens.
+
+1. In DevTools, on the **Activity Bar**, select the **Memory** tool.  If that tool isn't visible, click the **More tools** (![More tools icon](./js-runtime-images/more-tools-icon.png)) button.
+
+   ![Record Allocation sampling](./js-runtime-images/memory-allocation-sampling.png)
+
+1. Select the **Allocation sampling** option button.
+
+1. In the **Select JavaScript VM instance** section, if there is a worker on the page, you can select that as the profiling target.
+
+1. Click the **Start** button.
+
+1. On the webpage, perform actions that you want to investigate.  For example, click the **Create scattered objects** button in the **Example 3: Scattered objects** webpage.
+
+1. In the upper left of the **Memory** tool, click the **Stop heap profiling** (![The "Stop heap profiling" icon](./js-runtime-images/stop-recording-icon.png)) button.
+
+   A new profile is added in the **Sampling profiles** section in the **Profiles** sidebar.  The profile shows a breakdown of memory allocation by function.  The default view is **Heavy (Bottom Up)**, which displays the functions that allocated the most memory at the top:
+
+   ![Allocation sampling](./js-runtime-images/memory-allocation-sampling-heavy-bottom-up.png)
+
+
 <!-- ====================================================================== -->
 ## Record a Sampling Profile
```

Comment:
This article now has two sections that describe how to record a sample profile, at line 36, and line 64. We should choose one and remove the other one.

Reply:
merged 2 sections, improved